modules/profile.js

```diff
@@ -0,0 +1,39 @@
+const Profile = () => `
+
```

Comment:
This should look better without the empty line here. The empty lines can come immediately after you close tags.

Reply:
Thanks I will work on that